File: Assignment26/Program1.c

#include<stdio.h>
//1.Write a program which accept string from user and count number of capiatal characters

//Input Marvellous : "Marvellous Multi OS"

//Output 4

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//             Function Name          :    CountCapital
//             Description            :    Function to count Capital letters in string 
//             Intput Argument        :    Integer
//             Output Argument        :    Integer
//             Author                 :    Swati Vinayak Lahane 
//             Date                   :    19 November 2023
//             
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

int CountCapital(char *str)
{
    int iCnt = 0;
    //filter

   
   while(*str != '\0')
   {
       if((*str >= 'A') && (*str <= 'Z'))
       {
           iCnt++;
       }
       str++;
   }
   return iCnt;

}


int main()
{

    char arr[20];
    int iRet = 0;

    printf("Enter Stiring\n");
    scanf("%[^'\n']s",arr);

    iRet = CountCapital(arr);
    printf("Frequency  of capital character :%d\n",iRet);

    return 0;
    
}

File: Assignment32/Program1.java


import java.util.*;

//Problem on string
//1.Write a java program which accept string from user and and count Number of capital character


//Input : Marvellous Multi OS
//Output: 4

class Program1
{
    public static int CountCapital(String str)  
    {   
        int icnt = 0;
        int iCount = 0;
        char Arr[] = str.toCharArray();
        
        for(icnt =0; icnt < Arr.length; icnt++)
        {
           if((Arr[icnt] >= 'A') && (Arr[icnt] <= 'Z'))
           {
            iCount++;
           } 
            
        }
        return iCount;
    } 

    public static void main(String Args[])
    {
        Scanner sobj = new Scanner(System.in);
        String str = null;
        
        int iRet = 0;
        System.out.println("Enter String:");
        str = sobj.nextLine();
        
        
        iRet = CountCapital(str);
        
        System.out.println("Count of capital letters are:"+iRet);
         sobj.close();  

    }

}
